packages/Organisms/Grants/GrantsOverview.vue

```diff
@@ -71,7 +72,11 @@ export default {
               status = this.statusTerms.ongoing
             }
           } else {
-            status = this.statusTerms.closed
+            if (end) {
+              status = this.statusTerms.openingsoon
+            } else {
+              status = this.statusTerms.ongoing
```

Comment:
Hi @dylankelly I am not sure the logic here is right, in here, the now date is before start date, it should always be `openingsoon`, isn't it? Neither `onging` nor `closed`.

Reply:
ah yes your right, if it hasn't opened yet it is always opening soon. Have fixed.